src/Core/Transformers/JsonApi/Request/Serializers/Helpers/serializeResourceRelationships.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {HyralTask} task
+ *
+ * @returns {{type, id}}
+ */
+import serializeRelationTask from '../serializeRelationTask';
+
+export default function serializeResourceRelationships(task) {
+  return task.related
+    .reduce((serialized, relatedTask) => {
+      relatedTask.claim();
```

Comment:
waarom moet hij geclaimed worden bij serializen? dat wordt bij serializeResource niet gedaan

Reply:
Omdat in dit geval JsonApi een gerelateerde taak in de hoofdtaak gaat oppakken, Hyral hoeft deze taak dus niet meer los af te handelen